packages/common/client-utils/src/test/jest/tsconfig.cjs.json

```diff
@@ -1,14 +1,14 @@
 {
-	"extends": ["../../../../../../common/build/build-common/tsconfig.test.json"],
-	"include": ["./**/*"],
-	"references": [
-		{
-			"path": "../../..",
-		},
-	],
+	"extends": ["../../../../../../common/build/build-common/tsconfig.test.node16.json"],
```

Comment:
@DLehenbauer, I noticed the surrounding `[]` was removed in the base tsconfig.json file, but present in all of these others. I think it is fine to leave them be to get this merged.

Reply:
Weel, needs another fix per test failures; so I will clean these up too.